src/views/v2/day/event/date/featured-icon.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * View: Featured Icon.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/components/featured-icon.php
```

Comment:
```suggestion
 * [your-theme]/tribe/events/v2/day/event/date/featured.php
```

Reply:
Let's call all our templates "featured" rather than "featured-icon", some aren't just icons, but also with the "Featured" text.